tox.ini

```diff
@@ -82,4 +82,17 @@ pass_env =
     CI
 commands =
     poetry install --with integration
-    poetry run pytest -vv --tb native --log-cli-level=INFO -s {posargs} {[vars]tests_path}/integration
+    poetry run pytest -vv --tb native --log-cli-level=INFO -s {posargs} {[vars]tests_path}/integration/test_charm.py
+
+[testenv:integration-tls]
```

Comment:
**todo** Could we start using [this](https://github.com/canonical/kafka-k8s-operator/blob/main/tox.ini#L87) pattern for charm and tls? (maybe soon we will add more...)